packages/blockchain-api/src/config.ts

```diff
@@ -14,10 +14,33 @@ function getSecrets(deployEnv: string) {
   return envSecrets
 }
 
+export function getFirebaseAdminCreds(admin: any) {
+  // TODO: move project to celo-org-mobile
+  // until then, using serviceAccountKey for all envs
+  // tslint:disable-next-line: no-constant-condition
+  if (true /* DEPLOY_ENV === 'local' */) {
```

Comment:
Remove if (true) ? Or uncomment /* DEPLOY_ENV === 'local' */?

Reply:
I wanted to keep it this way so we can uncomment once we deploy it to `celo-org-mobile`.
Makes sense?